modules/core/src/adapters/deck-adapter.js

```diff
@@ -48,26 +48,28 @@ export default class DeckAdapter {
     this._getViewState = this._getViewState.bind(this);
     this._getLayers = this._getLayers.bind(this);
     this._applyScene = this._applyScene.bind(this);
+    this.enabled = false;
   }
 
   /**
    * @param {{ current: { deck: any; }; }} deckRef
    * @param {(ready: boolean) => void} setReady
    * @param {(nextTimeMs: number) => void} onNextFrame
    */
-  getProps(deckRef, setReady, onNextFrame) {
+  getProps(deckRef, setReady, onNextFrame, enabled) {
```

Comment:
Can you add the type documentation to the 2 or 3 interface changes?

I'd also like you to update the Markdown documentation for the DeckAdapter to explain them.